src/test/scala/com/atomist/rug/runtime/js/JavaScriptOperationFinderTest.scala

```diff
@@ -0,0 +1,82 @@
+package com.atomist.rug.runtime.js
```

Comment:
No need to do it in this PR, but I added ability to load a TypeScript side file from the class path. Makes it easier to edit the TS. For example,

` ``val jsed = TestUtils.editorInSideFile(this, "SimpleBanana.ts")```

Where the  TypeScript file is under test resources in the same package.